Back-end/prisma/schema.prisma

```diff
@@ -8,17 +8,18 @@ datasource db {
 }
 
 model User {
-  id          Int      @id @default(autoincrement())
-  username    String   @unique
+  id          Int        @id @default(autoincrement())
+  username    String     @unique
   password    String
   Headline    String
   FirstName   String
   LastName    String
   accountType String
-  date        DateTime @default(now())
+  date        DateTime   @default(now())
   Posts       Posts[]
   Token       Token[]
   Like        Like[]
+  Comments    Comments[]
```

Comment:
Any particular reason that you capitalize Comments/Like/Token but not username/id schema?

Reply:
I mostly capitalize the ones which are related to other models. I can change them if it is a better approach.

Reply:
Spoke in person that if I try to change the names, it forces me to delete my data from the database.